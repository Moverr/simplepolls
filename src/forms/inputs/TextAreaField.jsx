import React from 'react'

function TextAreaField(props){
    return <textarea {...props} >props.value</textarea>
}
export default TextAreaField;