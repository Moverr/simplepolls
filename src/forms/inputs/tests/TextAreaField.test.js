import React from 'react';
import ReactDOM from 'react-dom';
import { shallow } from 'enzyme';
 
import TextAreaField from '../TextAreaField';
 

describe('Testing Input FIeld', () => {
  it('should  render correctly', () => {    
    const props = {
        className:"inputfield",
        value:"text example",
        name:"name" 
           }

      const wrapper = shallow(<TextAreaField props />);
      expect(wrapper).toMatchSnapshot();
  });
})



 