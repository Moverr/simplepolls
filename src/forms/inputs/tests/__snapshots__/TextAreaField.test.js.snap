// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Input FIeld should  render correctly 1`] = `
<textarea
  props={true}
>
  props.value
</textarea>
`;
